monkey/infection_monkey/network/relay/utils.py

```diff
@@ -90,8 +89,7 @@ def send_remove_from_waitlist_control_message_to_relays(servers: Iterable[str]):
 
 
 def _send_remove_from_waitlist_control_message_to_relay(server: str):
```

Comment:
This should have been updated to accept the server as a `SocketAddress`. We should do this in another PR

Reply:
Change the logic. This function doesn't exist anymore.